<?xml version="1.0" encoding="utf-8"?>
<TestRun id="db97eb91-a295-48ea-a1f6-ca1e34209acf" name="tc@CISERVER 2021-02-27 11:41:46" runUser="CISERVER\tc" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2021-02-27T11:41:46.5907798+00:00" queuing="2021-02-27T11:41:46.5907806+00:00" start="2021-02-27T11:41:45.0345805+00:00" finish="2021-02-27T11:41:46.6080707+00:00" />
  <TestSettings name="default" id="b83354e5-da50-4430-81ea-b63963a20325">
    <Deployment runDeploymentRoot="tc_CISERVER_2021-02-27_11_41_46" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="15c5c332-1486-465c-b83f-ba36a9af6f78" testId="b03b56e8-d03d-329b-8fc8-46d37342c36b" testName="Test_Passes" computerName="CISERVER" duration="00:00:00.0001190" startTime="2021-02-27T11:41:46.0000000+00:00" endTime="2021-02-27T11:41:46.0000000+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="15c5c332-1486-465c-b83f-ba36a9af6f78" />
    <UnitTestResult executionId="7b3e2326-e1a5-4b09-bcad-670b37d4d875" testId="d572f632-bfaa-58e3-dd8e-dd281005c0ad" testName="Test_Fails" computerName="CISERVER" duration="00:00:00.0241200" startTime="2021-02-27T11:41:46.0000000+00:00" endTime="2021-02-27T11:41:46.0000000+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="7b3e2326-e1a5-4b09-bcad-670b37d4d875">
      <Output>
        <ErrorInfo>
          <Message>System.Exception : Ahhhhhhh!!!</Message>
          <StackTrace>   at TrxTests.NUnit.Tests.FailingTests.Test_Fails() in SomeDir\tyrannosaurus-trx\example-tests\TrxTests.NUnit.Tests\UnitTest1.cs:line 34</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="5b2a44dc-f038-47f0-a2a3-f9f011425fb5" testId="f8ae18fe-de42-54f1-84c0-083e30d104aa" testName="Test_Fails" computerName="CISERVER" duration="00:00:00.0006110" startTime="2021-02-27T11:41:46.0000000+00:00" endTime="2021-02-27T11:41:46.0000000+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="5b2a44dc-f038-47f0-a2a3-f9f011425fb5">
      <Output>
        <ErrorInfo>
          <Message>System.Exception : Ahhhhhhh!!!</Message>
          <StackTrace>   at TrxTests.NUnit.Tests.Tests.Test_Fails() in SomeDir\tyrannosaurus-trx\example-tests\TrxTests.NUnit.Tests\UnitTest1.cs:line 20</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="Test_Fails" storage="SomeDir\tyrannosaurus-trx\example-tests\trxtests.nunit.tests\bin\debug\netcoreapp3.1\trxtests.nunit.tests.dll" id="f8ae18fe-de42-54f1-84c0-083e30d104aa">
      <Execution id="5b2a44dc-f038-47f0-a2a3-f9f011425fb5" />
      <TestMethod codeBase="SomeDir\tyrannosaurus-trx\example-tests\TrxTests.NUnit.Tests\bin\Debug\netcoreapp3.1\TrxTests.NUnit.Tests.dll" adapterTypeName="executor://nunit3testexecutor/" className="TrxTests.NUnit.Tests.Tests" name="Test_Fails" />
    </UnitTest>
    <UnitTest name="Test_Passes" storage="SomeDir\tyrannosaurus-trx\example-tests\trxtests.nunit.tests\bin\debug\netcoreapp3.1\trxtests.nunit.tests.dll" id="b03b56e8-d03d-329b-8fc8-46d37342c36b">
      <Execution id="15c5c332-1486-465c-b83f-ba36a9af6f78" />
      <TestMethod codeBase="SomeDir\tyrannosaurus-trx\example-tests\TrxTests.NUnit.Tests\bin\Debug\netcoreapp3.1\TrxTests.NUnit.Tests.dll" adapterTypeName="executor://nunit3testexecutor/" className="TrxTests.NUnit.Tests.Tests" name="Test_Passes" />
    </UnitTest>
    <UnitTest name="Test_Fails" storage="SomeDir\tyrannosaurus-trx\example-tests\trxtests.nunit.tests\bin\debug\netcoreapp3.1\trxtests.nunit.tests.dll" id="d572f632-bfaa-58e3-dd8e-dd281005c0ad">
      <Execution id="7b3e2326-e1a5-4b09-bcad-670b37d4d875" />
      <TestMethod codeBase="SomeDir\tyrannosaurus-trx\example-tests\TrxTests.NUnit.Tests\bin\Debug\netcoreapp3.1\TrxTests.NUnit.Tests.dll" adapterTypeName="executor://nunit3testexecutor/" className="TrxTests.NUnit.Tests.FailingTests" name="Test_Fails" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="b03b56e8-d03d-329b-8fc8-46d37342c36b" executionId="15c5c332-1486-465c-b83f-ba36a9af6f78" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="d572f632-bfaa-58e3-dd8e-dd281005c0ad" executionId="7b3e2326-e1a5-4b09-bcad-670b37d4d875" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="f8ae18fe-de42-54f1-84c0-083e30d104aa" executionId="5b2a44dc-f038-47f0-a2a3-f9f011425fb5" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="3" executed="3" passed="1" failed="2" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>NUnit Adapter 3.15.1.0: Test execution started&#xD;
Running all tests in SomeDir\tyrannosaurus-trx\example-tests\TrxTests.NUnit.Tests\bin\Debug\netcoreapp3.1\TrxTests.NUnit.Tests.dll&#xD;
   NUnit3TestExecutor converted 3 of 3 NUnit test cases&#xD;
NUnit Adapter 3.15.1.0: Test execution complete&#xD;
</StdOut>
    </Output>
  </ResultSummary>
</TestRun>